What comes after "Settin up the Node-app ..."

I followed your instruction but still failed to run my first sample program.
I made a index.js file in the new directory (called test) and tried to run it using node index.js
I get the error module.js:440, throw err; .... Cannot find module 'zeropi'
I also don´t understand how the module is linked to the js-program. It would be great if you could explain the first steps to run the .js file a bit more detailed.

--- a/README.md
+++ b/README.md
@@ -80,9 +80,14 @@ This will detail the setup of your application. If you know how to do this alrea
 - Enter the directory: `cd MyApp`
 - Initialise npm in the app: `npm init`
 - Fill in all the details to suit (or hit enter until finished to get a basic setup)
+- Use npm to install the `zeropi` module: `npm install zeropi --save` (this will save the reference to the module in your package.json file)
 - Create an index.js (or whatever you want to name it)
 
 ## Basic Usage
+
+```
+npm install zeropi --save
+```
 
 ```javascript
 const ZeroPi = require('zeropi');

--- a/lib/core.js
+++ b/lib/core.js
@@ -9,12 +9,12 @@ class ZeroPiCore {
 
     // Serial port setup
     this.options = options || {};
-    this.options.debug = options.debug || false;
+    this.options.debug = this.options.debug || false;
     this.isOpen = false;
     this.selectors = {};
 
-    const path = options.path || '/dev/ttyAMA0';
-    const baudrate = options.baudrate || 115200;
+    const path = this.options.path || '/dev/ttyAMA0';
+    const baudrate = this.options.baudrate || 115200;
     this.serialPort = new SerialPort(path, { baudrate });
     this.serialPort.on('open', () => this.open());
 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "zeropi",
-  "version": "0.0.3",
+  "version": "0.0.4",
   "description": "A node 5.0.0+ module for running a zeropi",
   "main": "lib/zeropi.js",
   "scripts": {

